ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { TABLE } from "../../../../Constant/constant";
+
+export default function SortCursor(props) {
+  const [currentClickedCursor, setCurrentClickedCursor] = useState("");
+
+  const onCursorClick = (clickedCursorType,headerName) =>{
+    setCurrentClickedCursor(clickedCursorType);
+    props.handleSorting(clickedCursorType, headerName);
+  }
+
+  return (
+    <div>
+      <div
+        className={`up-arrow ${
+          currentClickedCursor === TABLE.ASCENDING ? "clicked-u" : ''
+        }`}
+        title={TABLE.ASCENDING}
+        onClick={() => {
+          onCursorClick(TABLE.ASCENDING, props.header);
+        }}
+      ></div>
+      <div
```

Comment:
These icons should be accessible. Instead of `DIV` we should use Button.

Reply:
Will work on it with a different ticket.

Reply:
Issue has been created